python/bolt-mcp/src/bolt_mcp/oauth/models.py

```
@@ -0,0 +1,63 @@
+    client_name = models.CharField(max_length=255, blank=True, default="")
+    redirect_uris = models.JSONField(default=list)
+    grant_types = models.JSONField(default=list)
+    scope = models.TextField(blank=True, default="")
+    token_endpoint_auth_method = models.CharField(max_length=64, default="none")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self) -> str:
+        return f"{self.client_name or 'client'} ({self.client_id})"
+
+
+class AuthorizationCode(models.Model):
```

Ruff (0.15.15)

[warning] Model does not define `__str__` method

(DJ008)